owasp-top10-2017-apps/a7/comment-killer/README.md

```diff
@@ -0,0 +1,58 @@
+# Comment-killer
+
+Comment-killer is a simple JS app that contains an example of multiple Cross-Site Scripting vulnerabilities and its main goal is to describe how a malicious user could exploit them on this purposefully vulnerable app.
+
+<img src="image/img1.png" alt="img1.png"/>
+
```

Comment:
Before getting into `What is Cross-Site Scripting?`, could please add an Index here?

You can use [this one](https://github.com/globocom/secDevLabs/blob/master/owasp-top10-2017-apps/a7/gossip-world/README.md#index) as a template.